site/app/controllers/admin/UsersController.php

```diff
@@ -355,7 +355,8 @@ public function updateUser($type = 'users') {
             $this->core->addSuccessMessage("User '{$user->getId()}' updated");
         }
         else {
-            if ($this->core->getQueries()->getSubmittyUser($_POST['user_id']) === null) {
+            $submitty_user = $this->core->getQueries()->getSubmittyUser($_POST['user_id']);
+            if ($submitty_user  === null) {
```

Comment:
```suggestion
            if ($submitty_user === null) {
```

Another small nitpick comment here.  It's interesting that PHPCS didn't pick this up--as far as I am aware it used to complain about stuff like this.  I'll look into it.